src/models.py

```diff
@@ -36,19 +31,21 @@ def check_password(self, password_to_check: str) -> bool:
         """Use this method to check User's password."""
         return self._get_password_hash(password_to_check) == self.password_hash
 
-    def create_user(self, username: str, password: str, session: scoped_session) -> None:
+    @classmethod
+    def create_user(cls, username: str, password: str) -> None:
```

Comment:
Since this method is actually creates a new object let's make it to return this object. Like we did it in `user.create_session()` method.

Yes now we don't use the returned object, but it's a good design to make "create_" methods return created object.

And we need to do the same thing for `Topic.create_topic()` and `Post.create_post()`